pkg/config/configuration.go

```diff
@@ -261,21 +260,41 @@ func (m MetricSpec) GetEnabled() bool {
 }
 
 // GetHTTPIncreasedCardinality returns true if increased cardinality is enabled for HTTP metrics
-func (m MetricSpec) GetHTTPIncreasedCardinality(log logger.Logger) bool {
+func (m MetricSpec) GetHTTPIncreasedCardinality() bool {
 	if m.HTTP == nil || m.HTTP.IncreasedCardinality == nil {
 		// The default is false
 		return false
 	}
 	return *m.HTTP.IncreasedCardinality
 }
 
+// GetHTTPPathNormalization returns the path normalization configuration for HTTP metrics
+func (m MetricSpec) GetHTTPPathNormalization() *PathNormalization {
+	if m.HTTP == nil || m.HTTP.PathNormalization == nil {
+		return nil
+	}
+	return &PathNormalization{
+		Enabled:      m.HTTP.PathNormalization.Enabled,
+		IngressPaths: m.HTTP.PathNormalization.IngressPaths,
+		EgressPaths:  m.HTTP.PathNormalization.EgressPaths,
+	}
```

Comment:
Couldn't this be just `return m.HTTP.PathNormalization`? Why creating a new object?

```suggestion
	if m.HTTP == nil {
		return nil
	}
	return m.HTTP.PathNormalization
```